ddtrace/span.py

```diff
@@ -210,20 +210,19 @@ def duration(self, value):
         self.duration_ns = int(value * 1e9)
 
     def finish(self, finish_time=None):
-        # type: (Optional[int]) -> None
+        # type: (Optional[float]) -> None
         """Mark the end time of the span and submit it to the tracer.
-        If the span has already been finished don't do anything
+        If the span has already been finished don't do anything.
 
-        :param int finish_time: The end time of the span in seconds.
-                                Defaults to now.
+        :param Optional[float] finish_time: The end time of the span, in seconds.
```

Comment:
We can probably exclude the type hint here since I think sphinx can retrieve it from the type hint above

Reply:
I wasn't sure so I left it. Thanks for pointing this out!